app/services/bulkrax/remove_relationships_for_importer.rb

```diff
@@ -68,6 +69,7 @@ def break_relationships!
       end
     end
 
+    # @todo(bjustice) - remove hyrax reference
```

Comment:
I've seen a few of these comments. is there more to do or can they be deleted?

Reply:
There is more to do that is not vital to this all working for importers. the idea is that once we test exporting and relationships we can come back and make the needed changes.

Reply:
agreeing to just have a comment here to clarify that if users want to add additional (or non hyrax) metadata here is the place to do it for collections.